fastlane/metadata/android/pl/title.txt

```diff
@@ -0,0 +1 @@
+Binary Eye
```

Comment:
If you want we can translate the app name to Polish too.
```suggestion
Binarne Oko
```

Reply:
Hm, I think that could interfere with findability, if you read about the app in an English context, for example 🤔